### DISCUSSION TITLE
nested labelProp in SelectElement

### DISCUSSION THREAD

**Discussion by hosseinjarrahi** [vueform/vueform#248] [Questions]
hey man, i have an array of objects in my items like below 

```js
[
  {
      nestedLabel: {
          realLabel: 'me'
     }
  }
]
```

in SelectElement is there a way to use nested LabelProp?
for example ```labelProp: 'nestedLabel.realLabel'```

**Comment by adamberecz** [comment]
Currently the only way would be to use an [async](https://vueform.com/reference/select-element#async) option list instead of endpoint and transform the received option list to have a flat structure.
